### PR TITLE
darts: implement new exercise

### LINKED ISSUE #1661: darts: implement new exercise
[Canonical-data](https://github.com/exercism/problem-specifications/blob/master/exercises/darts/canonical-data.json)

[Exercise description](https://github.com/exercism/problem-specifications/blob/master/exercises/darts/description.md)

### PR DESCRIPTION
**Pull request by GascaK**
Closes #1661
* Darts implementation
* Updated README.md

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1425,6 +1425,17 @@
...
     },
+    {
+      "slug": "darts",
+      "uuid": "cb581e2c-66ab-4221-9884-44bacb7c4ebe",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 2,
+      "topics": [
+          "algorithms",
+          "control_flow"
```
Comment (cmccandless) [inline config.json]: Topics should be taken from the [global topics list](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt); `control_flow` is not listed there.

I'm not sure `algorithms` can be applied here.

What about:
```JSON
"topics": [
	"conditionals",
	"math"
]
```

File: exercises/darts/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+For more detailed information about running tests, code style and linting,
+please see [Running the Tests](http://exercism.io/tracks/python/tests).
+
+## Source
+
+Inspired by an excersie created by a professor Della Paolera in Argentina
+
+## Submitting Incomplete Solutions
+
+It's possible to submit an incomplete solution so you can see how others have completed the exercise.
```
Comment (cmccandless) [inline exercises/darts/README.md]: Please use [`configlet generate`](https://github.com/exercism/configlet) to generate exercise READMEs.

File: exercises/darts/example.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from math import sqrt
+
+
+def Score(x, y):
```
Comment (cmccandless) [inline exercises/darts/example.py]:
```suggestion
def score(x, y):
```

File: exercises/darts/darts_test.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import unittest
+from darts import Score
```
Comment (cmccandless) [inline exercises/darts/darts_test.py]:
```suggestion
from darts import score
```

File: exercises/darts/darts.py [hunk 5]
```diff
@@ -0,0 +1,2 @@
+def Score(x, y):
```
Comment (cmccandless) [inline exercises/darts/darts.py]: Method names should start with lowercase.

```suggestion
def score(x, y):
```

File: config.json (hunk 1 above)
Comment (GascaK) [reply config.json]: I agree and have made changes.

File: exercises/darts/README.md (hunk 2 above)
Comment (GascaK) [reply exercises/darts/README.md]: The README was actually generated by configlet, Can you show me where this issue can be fixed?

File: exercises/darts/example.py (hunk 3 above)
Comment (GascaK) [reply exercises/darts/example.py]: Agreed and completed.

File: exercises/darts/darts_test.py (hunk 4 above)
Comment (GascaK) [reply exercises/darts/darts_test.py]: Changed to match score function.

File: exercises/darts/darts.py (hunk 5 above)
Comment (GascaK) [reply exercises/darts/darts.py]: Changed to match score function.

File: exercises/darts/README.md (hunk 2 above)
Comment (cmccandless) [reply exercises/darts/README.md]: My mistake; I noticed that there was not an empty list before the heading at line 18, but it appears that this is an issue with the template that has gone unnoticed.

**Comment by cmccandless** [comment]
Merged; thanks for working on this!

**Pull request merged**